docs/madr/decisions/032-zone-to-global-policy-sync.md

```diff
@@ -0,0 +1,185 @@
+path to Multizone and so can't be considered as good starting option when there are plans to scale across different AZs.
+
+Multizone mode has Global as a single source of truth for Kuma Policies. This can cause various operational challenges:
+* RBAC for teams to write to Global
+* GitOps can be tricky if team wants to keep policies in the same repo as code. For example, some users ended up
+  having a separate repo for Kuma policies. Application developers have to open PRs to this repo and get approval by SRE.
+  So instead of uncoupling and decentralization we got kind of "bottleneck" because of the way mesh syncs policies.
+* Converting Standalone to Zone CP and connecting to Global is not straightforward and requires manual policy syncing
+* k8s-native approach is to expect policies affect the cluster where they were applied
+* users have to use both kumactl and kubectl when Global is Universal and Zone is K8s
+* MeshGateway has to be applied on Global, while MeshGatewayInstances is created on Zone
+
+## Considered Options
+
+* Apply policies on Zone CP, scope is bounded to a single zone
+* Apply policies on Zone CP, no boundaries
+* Separate component that syncs policies from Zone to Global
+* Do nothing
+
+## Decision Outcome
+
+Apply policies on Zone CP, scope is bounded to a single zone. Optionally in the future we can implement a separate
+component that syncs policies from Zone to Global
+
+### Positive Consequences
+
+* Teams that are running Zones could be autonomous and apply policies for their services without access to Global
+* Store policies in the app repo for GitOps
+* Potential to create an automated Standalone to Zone conversion when all the policies stays on the cluster as is.
+  Application developers don't need to be aware that there was any standalone to multizone migration. From their perspective
+  the workflow stays the same. Standalone mode can be abolished, CP is Zone by default even if there is no connection to Global.
+* Policies take effect in the cluster where they were applied
+
+### Negative Consequences
+
+* Still have to use `kumactl` with access to Global when user wants to change client's outbounds in another Zone
+
+## Pros and Cons of the Options
+
+### Apply policies on Zone CP, scope is bounded to a single zone
+
+Users apply policies on Zone CP and Kuma automatically adds `kuma.io/zone` tag to the top-level targetRefs before syncing
+these policies to Global. For example, when user creates a policy in `us-east` zone:
+
+```yaml
+type: MeshTimeout
+mesh: mesh-1
+name: t-1
+labels: # not implemented today, requires Postgres schema 
+  kuma.io/managed-by: zone
+spec:
+  targetRef:
+    kind: MeshService
+    name: backend
+  to: [...]
+```
+
+Kuma automatically converts this to:
+
+```yaml
+type: MeshTimeout
+mesh: mesh-1
+name: t-1
+labels: # not implemented today, requires Postgres schema 
+  kuma.io/managed-by: zone
```

Comment:
should that be the zone name?

Reply:
We don't want to force users to create templates with `{{ zoneName }}` if they need pretty much the same policy for different zones. That was @johnharris85 suggestion, I kind of agree with it.

Reply:
Also, we're going to add `kuma.io/zone` tag automatically to the top-level targetRef.